Sources/StorageManager/FirebaseStorageManager.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  FirebaseStorageManager.swift
+//  
+//
+//  Created by Anna Münster on 28.06.22.
+//
+
+import Foundation
+import FirebaseStorage
+import Combine
+
+open class FirebaseStorageManager {
+    
+    public static func uploadFile(localFile: URL, path: String, fileName: String, fileType: FileType) -> AnyPublisher<Bool, StorageError> {
+        Future { promise in
+            
+            let storageRef = Storage.storage().reference(withPath: "\(path)/\(fileName).\(fileType.rawValue)")
+            
+            let taskReference = storageRef.putFile(from: localFile, metadata: nil) { metadata, error in
+                if let error = error {
+                    promise(.failure(.upload(error: error)))
+                }
+                promise(.success(true))
+            }
+            
+            taskReference.observe(.progress) { snapshot in
+                guard let progress = snapshot.progress?.fractionCompleted
+                
+                else { return }
```

Comment:
Is this weird line indentation from Github? If not, please correct it.

Reply:
Fixed